.gitignore

```diff
@@ -27,6 +27,7 @@ __pycache__
 /.cache
 /.testmondata
 /.hypothesis
+/.mypy_cache
```

Comment:
You don't need to split it up now, but stuff like this usually should go into a separate commit at least :wink: